client_app/src/App.js

```diff
@@ -1,14 +1,38 @@
-import {UpcomingShifts} from "./Shifts";
+import { BrowserRouter as Router, Routes, Route, Link } from "react-router-dom";
+
+import logo from "./Assets/gethelplogo.png";
+
+import { PastShifts, UpcomingShifts } from "./Shifts";
 import Shelters from "./Shelters";
+import VolunteerDashboard from "./VolunteerDashboard";
+import ShelterDashboard from "./ShelterDashboard";
+
 import "./App.css";
 
 function App() {
   return (
     <div>
-        {UpcomingShifts()};
-        {Shelters()};
+      <Router>
+        <div class="main-nav">
+          <Link to="/">
+            <img class="logo" src={logo} alt="GetHelp Logo" />
+          </Link>
+          <Link to="/shelters">Shelters</Link>
+          <Link to="/past-shifts">Previous Shifts</Link>
+          <Link to="/upcoming-shifts">Upcoming Shifts</Link>
+          <Link to="/shelter-dashboard">Shelter Dashboard</Link>
```

Comment:
```suggestion

```
I think we don't want to put shelter dashboard in this app, because it's a volunteer app (not shelter vacing app)